packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -1575,37 +1575,60 @@ integTest('skips notice refresh', withDefaultFixture(async (fixture) => {
 }));
 
 /**
- * Create a queue with a fresh name, redeploy orphaning the queue, then import it again
+ * Create a queue, orphan that queue, then import the queue.
+ *
+ * We want to test with a large template to make sure large templates can work with import.
  */
 integTest('test resource import', withDefaultFixture(async (fixture) => {
-  const outputsFile = path.join(fixture.integTestDir, 'outputs', 'outputs.json');
+  // GIVEN
+  const randomPrefix = randomString();
+  const uniqueOutputsFileName = `${randomPrefix}Outputs.json`; // other tests use the outputs file. Make sure we don't collide.
+  const outputsFile = path.join(fixture.integTestDir, 'outputs', uniqueOutputsFileName);
   await fs.mkdir(path.dirname(outputsFile), { recursive: true });
 
-  // Initial deploy
   await fixture.cdkDeploy('importable-stack', {
-    modEnv: { ORPHAN_TOPIC: '1' },
+    modEnv: { LARGE_TEMPLATE: '1', INCLUDE_SINGLE_QUEUE: '1', RETAIN_SINGLE_QUEUE: '1' },
     options: ['--outputs-file', outputsFile],
   });
 
   const outputs = JSON.parse((await fs.readFile(outputsFile, { encoding: 'utf-8' })).toString());
-  const queueName = outputs.QueueName;
-  const queueLogicalId = outputs.QueueLogicalId;
-  fixture.log(`Setup complete, created queue ${queueName}`);
+  fixture.log('Setup complete');
+
+  let queues: { [queueLogicalId: string]: { QueueUrl: string } } = {};
   try {
-    // Deploy again, orphaning the queue
+    // Write a resource mapping file based on the ID from step one, then run an import
+    const mappingFile = path.join(fixture.integTestDir, 'outputs', `${randomPrefix}Mapping.json`);
+    const fullStackName = fixture.fullStackName('importable-stack');
+    const queueUrl = outputs[fullStackName].QueueUrl;
+    const queueLogicalId = outputs[fullStackName].QueueLogicalId;
+    queues[queueLogicalId] = { QueueUrl: queueUrl };
```

Comment:
The way this comment is scoped makes it look like it's supposed to apply to the immediate block, which isn't the case. Can we move the comment outside the `try` so it's clear that it applies to the entire block? 

It would also be helpful if we numbered the steps, like 

1. Do something
2. Do something else